src/Files.App/Actions/IAction.cs

```diff
@@ -10,6 +10,8 @@ public interface IAction
 		RichGlyph Glyph => RichGlyph.None;
 
 		HotKey HotKey => HotKey.None;
+		HotKey SecondHotKey => HotKey.None;
```

Comment:
While you're at it, can you add support for a third hot key?

Reply:
It's done.